service/history/api/respondworkflowtaskcompleted/api.go

```diff
@@ -297,6 +297,12 @@ func (handler *WorkflowTaskCompletedHandler) Invoke(
 		if err != nil {
 			return nil, err
 		}
+
+		if handler.config.EnableReportedProblemsSearchAttribute(nsName) {
+			if err := ms.RemoveReportedProblemsSearchAttribute(); err != nil {
```

Comment:
I think there are some places where we force reset the workflow tasks' attempt to 1 (e.g. failover or buffered events ). Do we need to clear the SA as well in those places?